frontend/src/metabase/collections/components/CollectionContent/CollectionContentView.tsx

```diff
@@ -111,6 +112,9 @@ export const CollectionContentView = ({
   ] = useToggle(false);
   const [uploadedFile, setUploadedFile] = useState<File | null>(null);
 
+  // TODO: temp fix until we have the type field on effective_ancestors
```

Comment:
John is working on a fix for this, but our integration branch is currently broken without this, so if we can let this pass for now it would be appreciated!!